portfolio/src/main/webapp/index.html

```diff
@@ -1,49 +1,64 @@
 <!--Referenced to https://www.w3schools.com/html/default.asp-->
+<!--Referenced to https://www.w3schools.com/bootstrap/bootstrap_theme_me.asp-->
 
 <!DOCTYPE html>
 <html>
   <head>
     <meta charset="UTF-8">
     <title>Bernardo's Portfolio</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
I've heard that you can put <script> tags at the end of the body so that page load time is faster!

Reply:
Cool!! I'll try that